lua/genghis/init.lua

```diff
@@ -160,19 +160,50 @@ function M.chmodx()
 	cmd.edit()
 end
 
----@param opts? {trashLocation: string}
+---@param opts? {trashCmd: string, trashLocation: string}
 function M.trashFile(opts)
 	cmd.update { bang = true }
 	local home = os.getenv("HOME")
 	local oldFilePath = vim.api.nvim_buf_get_name(0)
 	local oldName = vim.fs.basename(oldFilePath)
 
+	local trashCmd
+	if opts ~= nil and opts.trashCmd ~= nil then
```

Comment:
just a nitpick, but since you might not know it: you can leave out `~= nil`, since `nil` is regarded as a "falsy" value in lua